src/main/java/com/digital/home/model/LabourDetailsKey.java

```diff
@@ -0,0 +1,37 @@
+package com.digital.home.model;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+
+public class LabourDetailsKey implements Serializable {
+
+    private long projectId;
+    private Date transactionDate;
+    private long labourTypeId;
+
+    public LabourDetailsKey() {
```

Comment:
When you don't have set methods, no need to have empty constructor

Reply:
Removed the empty constructor